src/Dockerfile

```diff
@@ -11,7 +11,7 @@ COPY . $WORKDIR
 # installing dependencies to build package
 RUN pip install . -t python
 
-# change 4573439369
+# change 4573439355693344
```

Comment:
I believe using the hash sum of the source code as trigger would be more ideal. This approach here seems error prone as one might forget to update the line

Reply:
true, this would require a change to the terraform module that does this, right?

Reply:
since this a change outside the repo and won't require any changes here once it's done, I just created a separate jira item to address this: https://gfw.atlassian.net/browse/GTC-1250